### PR TITLE
References api-priority-and-fairness slack channel in the docs

### PR DESCRIPTION
**Pull request by yue9944882**

### CODE REVIEW

File: content/en/docs/concepts/cluster-administration/flow-control.md [hunk 1]
```diff
@@ -514,5 +514,5 @@ When you enable the API Priority and Fairness feature, the kube-apiserver serves
 For background information on design details for API priority and fairness, see
 the [enhancement proposal](https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/20190228-priority-and-fairness.md).
 You can make suggestions and feature requests via [SIG API
-Machinery](https://github.com/kubernetes/community/tree/master/sig-api-machinery).
+Machinery](https://github.com/kubernetes/community/tree/master/sig-api-machinery) or our [slack channel](http://kubernetes.slack.com/messages/api-priority-and-fairness).
```
Comment (sftim) [inline content/en/docs/concepts/cluster-administration/flow-control.md:517]: nit:
```suggestion
Machinery](https://github.com/kubernetes/community/tree/master/sig-api-machinery) or the SIG's
[Slack channel](http://kubernetes.slack.com/messages/api-priority-and-fairness).
```

**Review by sftim — COMMENTED** [review]
/lgtm

File: content/en/docs/concepts/cluster-administration/flow-control.md (hunk 1 above)
Comment (yue9944882) [reply content/en/docs/concepts/cluster-administration/flow-control.md:517]: we dont have a dedicated sig yet. the feature is majorly maintained by api-machinery but also involves effort from scheduling and scalability. does that make sense?

**Comment by yue9944882** [comment]
@sftim can we get this merged? my mailbox is filled by the letters looking for support, it will be great if users can reach out to the slack channel in the first place..

**Review by irvifa — COMMENTED** [review]
/lgtm

File: content/en/docs/concepts/cluster-administration/flow-control.md [hunk 2]
```diff
@@ -527,5 +527,5 @@ When you enable the API Priority and Fairness feature, the kube-apiserver serves
 
 For background information on design details for API priority and fairness, see
 the [enhancement proposal](https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/20190228-priority-and-fairness.md).
-You can make suggestions and feature requests via [SIG API
-Machinery](https://github.com/kubernetes/community/tree/master/sig-api-machinery).
+You can make suggestions and feature requests via [SIG API Machinery](https://github.com/kubernetes/community/tree/master/sig-api-machinery) 
+or the feature's [slack channel](http://kubernetes.slack.com/messages/api-priority-and-fairness).
```
Comment (sftim) [inline content/en/docs/concepts/cluster-administration/flow-control.md:531]: nit:
```suggestion
or the feature's [Slack channel](http://kubernetes.slack.com/messages/api-priority-and-fairness).
```

**Review by sftim — COMMENTED** [review]
/approve
Thanks!
/sig api-machinery

**Pull request merged**